docs/api-reference-1/frontity.md

```diff
@@ -124,11 +136,10 @@ const { state, actions, libraries } = useConnect();
 
 It's a React hook that returns the Frontity state, allowing the component to consume `state`, `actions` and `libraries` in components without passing them as props.
```

Comment:
Could you please add a warning here so people understand that they still need to use `connect`?

You also indicate that `connect` improves the performance (so people don't ask why they need to also add connect`):

- It optimizes your components with memo, so they won't re-render whenever a parent component re-renders.
- It makes them reactive, so they will re-render when the parts of `state` they use are changed.

Reply:
Done! 👍